test/runtests.sh

```diff
@@ -7,7 +7,7 @@ then
 fi
 
 # delete output from previous run
-if [ ! -d "../actual.txt" ]
+if [ -e "./actual.txt" ]
```

Comment:
Oh right, it's precisely because this needs to be run in the `test` directory. Whoops.